src/metabase/pulse/render/body.clj

```diff
@@ -112,12 +120,14 @@
 (s/defn ^:private prep-for-html-rendering
   "Convert the query results (`cols` and `rows`) into a formatted seq of rows (list of strings) that can be rendered as
   HTML"
-  [timezone-id :- (s/maybe s/Str) card {:keys [cols rows]} bar-column max-value column-limit]
-  (let [remapping-lookup (create-remapping-lookup cols)
-        limited-cols (take column-limit cols)]
-    (cons
-     (query-results->header-row remapping-lookup card limited-cols bar-column)
-     (query-results->row-seq timezone-id remapping-lookup limited-cols (take rows-limit rows) bar-column max-value))))
+  ([timezone-id card data column-limit]
+   (prep-for-html-rendering timezone-id card data nil nil nil column-limit))
```

Comment:
There were a couple `nil nil`s scattered around, especially in tests, and adding `min-value` made it `nil nil nil`, so I decided it was cleaner to just overload it.